crates/node_binding/test.js

```diff
@@ -0,0 +1,21 @@
+const { formatDiagnostic } = require("./binding")
```

Comment:
this file is used only for local testing? or should we run real unit test for it

Reply:
This is for local testing only. Let me remove this.